BUILD

```diff
@@ -1,11 +1,22 @@
+#
+#  Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 #Simple Bazel build file that will be replaced with the actual fuzz target rule functionality.
 load("@rules_cc//cc:defs.bzl", "cc_binary")
 
-cc_binary(
-    name = "hello_world",
-    srcs = ["hello_world.cc"],
-)
-
 cc_test(
     name = "dummy_test",
     srcs = ["dummy_test.cc"],
```

Comment:
Remove this test too? Now that we have real fuzz targets to test, we no longer need the dummy code IMO.

Reply:
I deleted this dummy test before, but got github Actions bazel_test error, as it requires at least one test to run. For those fuzz target examples, they will return errors( e.g, oom), too. Maybe we keep this test until creating real test files?